Sources/Testing/Testing.docc/Expectations.md

```diff
@@ -10,16 +10,60 @@ See https://swift.org/LICENSE.txt for license information
 See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 -->
 
-Check for expected values and outcomes in tests.
+Check for expected values, outcomes, and events in tests.
```

Comment:
Events have special meaning to swift-testing and `#expect()` does not check for them.

Reply:
I agree, but `Confirmation` is in this API collection (see Topics) and `Confirmation` does check for events.

Reply:
How about:

> Check for expected values, outcomes, and asynchronous events in tests.

Since `confirmation()` should really only be used for testing out-of-band stuff.

Reply:
Mind making this change?